eventq.gemspec

```diff
@@ -26,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec'
   spec.add_development_dependency 'shoulda-matchers'
-  spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'simplecov', '< 0.18.0'
```

Comment:
Do we need this version lock?

Reply:
The version lock was a workaround to solve for an issue that came about when we switched to Github Actions. Kindly follow through the link below to get more context:
https://sageone.slack.com/archives/C02D3UFKJEN/p1630572853012700